### PR TITLE
fix contact form

### PR DESCRIPTION
**Pull request by evykassirer**
addresses #22 and #21

### CODE REVIEW

**Comment by arshiamufti** [comment]
i'm not going to be able to deploy this locally to see how it looks during work hours--can you upload a screenshot? if you can't i'll just do it when i get home

**Comment by evykassirer** [comment]
same :p can't deploy at work

tonight is totally fine

File: _pages/coc.md [hunk 1]
```diff
@@ -74,7 +74,7 @@ communications pertaining to community business.
 
 ## Contact Us ##
 
-We'll keep your message and contact information private.
+We won't share your contact information or the contents of your message with anyone other than conference executives.
 
 <form action="https://formspree.io/uwstarcon@gmail.com" method="POST">
```
Comment (arshiamufti) [inline _pages/coc.md]: you could also set `cols=55` or some other value to achieve the same effect

Comment (evykassirer) [inline _pages/coc.md]: ah cool I didn't know about that

does it work on mobile sized screens too?

**Pull request merged**